cnf/make.conf.example

```diff
@@ -111,24 +111,24 @@
 #     will protect the default locations of DISTDIR and PKGDIR, but users are
 #     warned that any other locations inside PORTDIR are not necessarily safe
 #     for data storage.
-#PORTDIR=/usr/portage
+#PORTDIR=/var/db/repos/gentoo
 #
 # DISTDIR is where all of the source code tarballs will be placed for
 #     emerges. After packages are built, it is safe to remove any and
 #     all files from this directory since they will be automatically
 #     fetched on demand for a given build. If you would like to
 #     selectively prune obsolete files from this directory, see
 #     eclean from the gentoolkit package. Note that locations under
-#     /usr/portage are not necessarily safe for data storage. See the
-#     PORTDIR documentation for more information.
-#DISTDIR=/usr/portage/distfiles
+#     /var/db/repos/gentoo are not necessarily safe for data storage.
+#     See the PORTDIR documentation for more information.
+#DISTDIR=/var/cache/distfiles
 #
```

Comment:
I believe it is unlikely that anyone would change the DISTDIR location to a subdir of /var/db/repos/gentoo. So the following two sentences are more confusing than helpful, and IMHO should be deleted entirely:
"Note that locations under /var/db/repos/gentoo are not necessarily safe for data storage. See the PORTDIR documentation for more information."

Reply:
done